src/main/java/de/tum/in/www1/artemis/service/ExerciseService.java

```diff
@@ -777,7 +778,11 @@ public void notifyAboutExerciseChanges(Exercise originalExercise, Exercise updat
         }
         else if (originalExercise.isExamExercise() && !StringUtils.equals(originalExercise.getProblemStatement(), updatedExercise.getProblemStatement())) {
             User instructor = userRepository.getUser();
-            this.examLiveEventsService.createAndSendProblemStatementUpdateEvent(updatedExercise, notificationText, instructor);
+            Exam exam = originalExercise.getExam();
+            // notify the student only if the exam is started
+            if (exam.isStarted()) {
```

Comment:
This check should be before comparing the problem statements as this comparison is more complex.